Libraries/LibELF/DynamicObject.cpp

```diff
@@ -160,6 +160,8 @@ void DynamicObject::parse()
         case DT_NEEDED:
             // We handle these in for_each_needed_library
             break;
+        case DT_RPATH:
```

Comment:
Pretty odd that the compiler/linker is generating DT_RPATH dynamic headers when built with macos? I would expect the configure flags to the gcc cross-compiler and binutils would be the same.

Should probably handle DT_RPATH and DT_RUNPATH in the same way. The proper way to implement this tag would be.. creating a list of strings for each string pointed to by entries in that section, and exposing it to the DynamicLoader for use in when it loads dependencies?

Anyway, I think if we want the "unblock MacOS" patch instead of the real impl it should look something like:
```
        case DT_RPATH:
        case DT_RUNPATH:
            // FIXME: Implement RPATH/RUNPATH handling in DynamicLoader
            break;
```